style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+   color: blueviolet;
```

Comment:
- [ ] About the Stylelint issue I think the problem is because it expects 2 spaces of indentation for better practices and this line contains more than 2 spaces. You can see these issues running `npx stylelint "**/*.{css,scss}"` locally :wink:
![image](https://user-images.githubusercontent.com/81522435/143045341-8c6bce50-0774-42be-b8f9-472bcaf72aba.png)

Reply:
It was due to the problem in my vscode but I have fix the issue here in the Github.